gui/src/components/OnboardingWizard/OnboardingWizard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Button, Stack, Typography } from "@mui/material";
+import { invoke } from "@tauri-apps/api/tauri";
+import { useState } from "react";
+
+import { useKeyPress } from "../../hooks";
+import { useWizardForm } from "../../store/wizard";
+import { Logo } from "../Logo";
+import { OnboardingCarousel } from "./OnboardingCarousel";
+import { steps } from "./Steps";
+
+interface Props {
+  closeOnboarding: () => void;
+}
+
+export function OnboardingWizard({ closeOnboarding }: Props) {
+  const [activeStep, setActiveStep] = useState(0);
+  const { alchemyApiKey } = useWizardForm();
```

Comment:
This feels like unnecessary complexity. What about reading directly from the settings with `  const { data: general, mutate } = useInvoke<GeneralSettings>("settings_get");`?

Reply:
That makes sense to read the initial value. I created the store to save the user inputs in case the user moves back and forth between the steps, and only apply these changes when the wizard is closed

Reply:
I added this data read and got rid of the store, the form data is now being stored in a state variable in `OnboardingWizard`